src/test/java/com/easyrestaurant/tests/ModeratorPanelPageTests.java

```diff
@@ -0,0 +1,55 @@
+package com.easyrestaurant.tests;
+
+import io.qameta.allure.Allure;
+import io.qameta.allure.Description;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+import com.easyrestaurant.core.TestUtils;
+
+public class ModeratorPanelPageTests extends TestUtils {
+
+
+    @Parameters({"username", "password"})
```

Comment:
When I ran tests I had this error message: " Parameter 'username' is required by @Test on method canApproveRestaurant but has not been marked @Optional or defined
in /home/desi/.cache/JetBrains/IdeaIC2022.3/temp-testng-customsuite.xml"

Reply:
The tests are refactored, please try to run them from AllTests.xml.
If you get the same error "Parameter 'username' is..." try to run the test from the separate test suite before run the AllTests.xml file.